src/main/java/org/minions/devfund/miguelcalderon/WhackAMole.java

```diff
@@ -0,0 +1,186 @@
+package org.minions.devfund.miguelcalderon;
+
+import java.util.Random;
+import java.util.Scanner;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Scanner'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)